app/controllers/invitations_controller.rb

```diff
@@ -36,7 +36,7 @@ def update
       return redirect_to organisation_path(organisation), alert: "Invitation not found."
     end
 
-    if @invitation.pending?
+    if @invitation.pending? || @invitation.accepted?
```

Comment:
If the user has accepted then they're a member so have progressed the invitation so this is NOOP